woocommerce/woocommerce-gutenberg-products-block
# Combobox patch to prevent the suggestion list being left open after browser autocomplete

This PR applies the patch from https://github.com/WordPress/gutenberg/pull/34090 to our local version of the package.

### How to test the changes in this Pull Request:

1. npm install so the patch is applied, build
2.  Go to checkout
3. Autocomplete the fields and confirm the combobox is not left hanging open
